README.rst

```diff
@@ -13,6 +13,31 @@ Python standard ``hasattr`` is considered harmful (within Plone projects).
 The (hidden) problem with ``hasattr`` is that it swallows exceptions,
 which in your normal business logic you really don't want to.
 
+Take, for instance, the following code:
+
+.. code-block:: python
+
+    >>> class Foo(object):
+    ...     @property
+    ...     def my_attr(self):
+    ...         raise ValueError('nope, nope, nope')
+    ...
+    >>> bar = Foo()
+    >>> bar.my_attr
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+      File "<stdin>", line 4, in my_attr
+    ValueError: nope, nope, nope
+    >>> hasattr(Foo, 'my_attr')
+    True
+    >>> hasattr(bar, 'my_attr')
+    False
+    >>> getattr(bar, 'my_attr', None)
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+      File "<stdin>", line 4, in my_attr
+    ValueError: nope, nope, nope
+
```

Comment:
Could you move this chunk after the next line? So is at the end of the section just above the note about p.r.codeanalysis? If not the line that is now (in your branch) below this chunk is completely out of context.

Thanks!